container/container.go

```diff
@@ -441,14 +450,15 @@ func getResource(name string, taskInfo *mesos.TaskInfo) *mesos.Resource {
 	return nil
 }
 
-// Prefix used to name Docker containers in order to distinguish those
-// created by Mesos from those created manually.
-const DockerNamePrefix = "mesos-"
-
 func GetContainerName(taskId *mesos.TaskID) string {
 	// unique uuid based on the TaskId
 	containerUUID := uuid.NewSHA1(uuid.NIL, []byte(*taskId.Value))
-	return DockerNamePrefix + containerUUID.String()
+
+	// Note: We can add another dockerNameSeparator after the UUID field
+	// if we want to add more custom string data after it
+	// Example: dockerNamePrefix + custom_string + dockerNameSeparator + ContainerUUID + dockerNameSeparator + another_custom_string
+	// Details here: https://github.com/apache/mesos/blob/3446e0a43eb16151d0f467e7034b0ca6baefda4a/src/slave/containerizer/docker.cpp#L151
+	return dockerNamePrefix + *taskId.Value + dockerNameSeparator + containerUUID.String()
```

Comment:
Like it! Btw, any reason why you didn't use [fmt.Sprintf](https://golang.org/pkg/fmt/#Sprintf) here to concatenate the returned `string`?

Reply:
Because there are no interleaving characters in the format. It would be only `%s%s%s%s`, which is rather pointless / harder to read.